cmd/node/node.go

```diff
@@ -479,6 +480,7 @@ func apiRunOptsFromCLIFlags() *api.RunOptions {
 	if *enableMetaMaskAPI {
 		if *buildExtendedApi {
 			opts.EnableMetaMaskAPI = *enableMetaMaskAPI
+			opts.EnableMetaMaskAPILog = *enableMetaMaskAPILog
```

Comment:
where is it used exactly? what is it for?

Reply:
This option enables or disables metamask API request logging. Take a look
https://github.com/wavesplatform/gowaves/blob/0633158232b01d19b06376ab6e81de15f2efe9e6/pkg/api/routes.go#L139-L150